src/features/share/asImage/ShareAsImage.tsx

```diff
@@ -123,22 +124,26 @@ interface ShareAsImageProps {
 export default function ShareAsImage({ data, header }: ShareAsImageProps) {
   const presentToast = useAppToast();
 
-  const [hideUsernames, setHideUsernames] = useState(false);
-  const [hideCommunity, setHideCommunity] = useState(false);
-  const [includePostDetails, setIncludePostDetails] = useState(
-    !("comment" in data),
-  );
```

Comment:
@aeharding I guess this was the cause of the issue, and its not browser dependent. I tried it in incognito window in edge and its reproducing

Reply:
I believe I have fixed the issue in my latest commit: [`c8af557` (#1581)](https://github.com/aeharding/voyager/pull/1581/commits/c8af557fdc3039b477eea217cd241da755af2b4e)

let me know if there's anything else